### PR TITLE
[hi] Localize community page

### PR DESCRIPTION
**Pull request by harish343**
Fixes : 39663

New feature has been added communtiy page in hindi please review it and ping me if any changes had to be made on this repository.
Fixed localize of alt and Title

### CODE REVIEW

**Comment by harish343** [comment]
Hey @sftim can you review this.

**Comment by sftim** [comment]
> Hey @sftim can you review this.

It will take a while! I'd have to learn Hindi.

File: content/hi/community/_index.html [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+  <div class="community-nav-item">
+    <a href="#news">समाचार</a>
+  </div>
+  <div class="community-nav-item">
+    <a href="/releases">रिलीज़ेज़</a>
+  </div>
+</div>
+
+<div class="community-section" id="gallery">
+  <img src="/images/community/kubernetes-community-final-03.jpg" alt="कुबेरनेट्स कॉन्फ्रेंस गैलरी फोटो" class="community-gallery-desktop">
```
Comment (Babapool) [inline content/hi/community/_index.html:48]:
```suggestion
  <img src="/images/community/kubernetes-community-final-03.jpg" alt="कुबेरनेट्स सम्मेलन की गैलरी तस्वीरें" class="community-gallery-desktop">
```

File: content/hi/community/_index.html [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+  <img src="/images/community/kubernetes-community-final-04.jpg" alt="कुबेरनेट्स कॉन्फ्रेंस गैलरी फोट" class="community-gallery-desktop">
```
Comment (Babapool) [inline content/hi/community/_index.html:49]:
```suggestion
  <img src="/images/community/kubernetes-community-final-04.jpg" alt="कुबेरनेट्स सम्मेलन की गैलरी तस्वीरें" class="community-gallery-desktop">
```

File: content/hi/community/_index.html [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+  <img src="/images/community/kubernetes-community-final-05.jpg" alt="कुबेरनेट्स कॉन्फ्रेंस गैलरी फोट" class="community-gallery-desktop">
```
Comment (Babapool) [inline content/hi/community/_index.html:50]:
```suggestion
  <img src="/images/community/kubernetes-community-final-05.jpg" alt="कुबेरनेट्स सम्मेलन की गैलरी तस्वीरें" class="community-gallery-desktop">
```

File: content/hi/community/_index.html [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+  <img src="/images/community/kubernetes-community-04-mobile.jpg" alt="कुबेरनेट्स कॉन्फ्रेंस गैलरी फोट" class="community-gallery-mobile">
```
Comment (Babapool) [inline content/hi/community/_index.html:51]:
```suggestion
  <img src="/images/community/kubernetes-community-04-mobile.jpg" alt="कुबेरनेट्स सम्मेलन की गैलरी तस्वीरें" class="community-gallery-mobile">
```

File: content/hi/community/_index.html [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+</div>
+
+<div class="community-section" id="values">
+  <h2>समुदाय के मूल्य</h2>
+  <p>कुबेरनेट्स समुदाय के मूल्य प्रोजेक्ट की निरंतर सफलता के लिए महत्वपूर्ण होते हैं।<br class="optional"/>
```
Comment (Babapool) [inline content/hi/community/_index.html:56]:
```suggestion
  <p>कुबेरनेट्स समुदाय जिन मूल्यों के लिए खड़ा है, वे परियोजना की निरंतर सफलता की कुंजी हैं।<br class="optional"/>
```

File: content/hi/community/_index.html [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+  ये सिद्धांत कुबेरनेट्स प्रोजेक्ट के हर पहलू को निर्देशित करते हैं।</p>
```
Comment (Babapool) [inline content/hi/community/_index.html:57]:
```suggestion
  ये सिद्धांत कुबेरनेट्स पपरियोजना के हर पहलू का मार्गदर्शन करते हैं।</p>
```

File: content/hi/community/_index.html [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+  <a href="https://www.kubernetes.dev/community/values/" class="community-cta-button">
+    <span class="community-cta">अधिक पढ़ें</span>
+  </a>
+</div>
+
+<div class="community-section" id="conduct">
+  <h2>आचरण संहिता</h2>
+  <p>कुबेरनेट्स समुदाय मूल्यों में सम्मान और समावेश शामिल हैं, और सभी इंटरैक्शन में आचरण संहिता का पालन करते हुए होते हैं।</p>
```
Comment (Babapool) [inline content/hi/community/_index.html:65]:
```suggestion
  <p>कुबेरनेट्स समुदाय मूल्यों में सम्मान और समावेश को महत्व देता, और सभी इंटरैक्शन में आचरण संहिता को लागू करता हैं।</p>
```

File: content/hi/community/_index.html [hunk 8]
```diff
@@ -0,0 +1,175 @@
...
+  <p>यदि आप किसी इवेंट या मीटिंग, <a href="#slack">स्लैक</a> या किसी अन्य संचार माध्यम में आचरण संहिता का उल्लंघन करते हुए देखते हैं, तो <a href="mailto:conduct@kubernetes.io">conduct@kubernetes.io</a> पर कुबेरनेट्स आचार संहिता समिति से संपर्क करें। सभी रिपोर्ट को गोपनीय रखा जाता है। आप गिटहब पर कुबेरनेट्स समुदाय रिपॉजिटरी में <a href="https://github.com/kubernetes/community/tree/master/committee-code-of-conduct">समिति के बारे में</a> पढ़ सकते हैं।</p>
```
Comment (Babapool) [inline content/hi/community/_index.html:66]:
```suggestion
  <p>यदि आप किसी इवेंट या मीटिंग, <a href="#slack">स्लैक</a> या किसी अन्य संचार माध्यम में आचरण संहिता का उल्लंघन करते हुए देखते हैं, तो <a href="mailto:conduct@kubernetes.io">conduct@kubernetes.io</a> पर कुबेरनेट्स आचार संहिता समिति से संपर्क करें। सभी रिपोर्ट को गोपनीय रखा जाता है। आप GitHub पर कुबेरनेट्स समुदाय रिपॉजिटरी में <a href="https://github.com/kubernetes/community/tree/master/committee-code-of-conduct">समिति के बारे में</a> पढ़ सकते हैं।</p>
```

**Review by Babapool — COMMENTED** [review]
Some suggestions from my side, hope you find them useful. Thanks

**Comment by Babapool** [comment]
@YashPimple, a friendly reminder from my end. I just wanted to confirm that since there have been no updates on this PR are you still working on this PR? Thanks

**Comment by harish343** [comment]
Hey @Babapool can you tell me how to proceed further

File: content/hi/community/_index.html [hunk 9]
```diff
@@ -0,0 +1,175 @@
...
+---
+<img
+  id="banner"
+  srcset="/images/community/kubernetes-community-final-02.jpg 1500w, /images/community/kubernetes-community-02-mobile.jpg 900w"
+  sizes="(max-width: 900px) 900px, (max-width: 1920px) 1500px"
+  src="/images/community/kubernetes-community-final-02.jpg"
+  alt="कुबेरनेट्स सम्मेलन तस्वीर">
+
+<div class="community-section" id="introduction">
+  <p>कुबेरनेट्स समुदाय - उपयोगकर्ता, योगदानकर्ता और हम सभी ने साथ मिलकर बनाई हुई संस्कृति - इस ओपन सोर्स प्रोजेक्ट के उच्चाकांक्षी उदय के सबसे बड़े कारणों में से एक है। हमारी संस्कृति और मूल्य तब तक बढ़ते रहते हैं जब तक प्रोजेक्ट और हम इस पर काम करने के तरीकों में सुधार करते रहें। हम सभी संयुक्त रूप से प्रोजेक्ट और हम इस पर काम करने के तरीकों को लगातार सुधारने के लिए काम करते हैं।</p>
```
Comment (aj11anuj) [inline content/hi/community/_index.html:15]:
```suggestion
  <p>कुबेरनेट्स समुदाय - उपयोगकर्ता, योगदानकर्ता और हम सभी ने साथ मिलकर बनाई हुई संस्कृति - इस ओपन सोर्स प्रोजेक्ट के उच्चाकांक्षी उदय के सबसे बड़े कारणों में से एक है। हमारी संस्कृति और मूल्य बढ़ते और बदलते रहते हैं जैसे की परियोजना बढ़ती है और बदलती है। हम सभी परियोजना के निरंतर सुधार और उस पर काम करने के तरीकों की दिशा में मिलकर काम करते हैं।</p>
```

File: content/hi/community/_index.html [hunk 10]
```diff
@@ -0,0 +1,175 @@
...
+  </div>
+  <div class="community-nav-item">
+    <a href="/releases">रिलीज़ेज़</a>
+  </div>
+</div>
+
+<div class="community-section" id="gallery">
+  <img src="/images/community/kubernetes-community-final-03.jpg" alt="कुबेरनेट्स सम्मेलन की गैलरी तस्वीरें" class="community-gallery-desktop">
+  <img src="/images/community/kubernetes-community-final-04.jpg" alt="कुबेरनेट्स सम्मेलन की गैलरी तस्वीरें" class="community-gallery-desktop">
+  <img src="/images/community/kubernetes-community-final-05.jpg" alt="कुबेरनेट्स कॉन्फ्रेंस गैलरी फोट" class="community-gallery-desktop">
```
Comment (aj11anuj) [inline content/hi/community/_index.html:50]:
```suggestion
<div class="community-section" id="gallery">
  <img src="/images/community/kubernetes-community-final-03.jpg" alt="कुबेरनेट्स सम्मेलन की तस्वीरें" class="community-gallery-desktop">
  <img src="/images/community/kubernetes-community-final-04.jpg" alt="कुबेरनेट्स सम्मेलन की तस्वीरें" class="community-gallery-desktop">
  <img src="/images/community/kubernetes-community-final-05.jpg" alt="कुबेरनेट्स कॉन्फ्रेंस की तस्वीरें" class="community-gallery-desktop">
```

File: content/hi/community/_index.html [hunk 11]
```diff
@@ -0,0 +1,175 @@
...
+  <img src="/images/community/kubernetes-community-04-mobile.jpg" alt="कुबेरनेट्स सम्मेलन की गैलरी तस्वीरें" class="community-gallery-mobile">
```
Comment (aj11anuj) [inline content/hi/community/_index.html:51]:
```suggestion
  <img src="/images/community/kubernetes-community-04-mobile.jpg" alt="कुबेरनेट्स सम्मेलन की तस्वीरें" class="community-gallery-mobile">
```

File: content/hi/community/_index.html [hunk 12]
```diff
@@ -0,0 +1,175 @@
...
+</div>
+
+<div class="community-section" id="values">
+  <h2>समुदाय के मूल्य</h2>
+  <p>कुबेरनेट्स समुदाय जिन मूल्यों के लिए खड़ा है, वे परियोजना की निरंतर सफलता की कुंजी हैं।<br class="optional"/>
```
Comment (aj11anuj) [inline content/hi/community/_index.html:56]:
```suggestion
  <p>कुबेरनेट्स समुदाय जिन मूल्यों पे खड़ा है, वे प्रोजेक्ट की निरंतर सफलता की कुंजी हैं।<br class="optional"/>
```

File: content/hi/community/_index.html [hunk 13]
```diff
@@ -0,0 +1,175 @@
...
+  ये सिद्धांत कुबेरनेट्स पपरियोजना के हर पहलू का मार्गदर्शन करते हैं।</p>
```
Comment (aj11anuj) [inline content/hi/community/_index.html:57]:
```suggestion
  ये सिद्धांत कुबेरनेट्स प्रोजेक्ट के हर पहलू का मार्गदर्शन करते हैं।</p>
```

File: content/hi/community/_index.html [hunk 14]
```diff
@@ -0,0 +1,175 @@
...
+  <a href="https://www.kubernetes.dev/community/values/" class="community-cta-button">
+    <span class="community-cta">अधिक पढ़ें</span>
+  </a>
+</div>
+
+<div class="community-section" id="conduct">
+  <h2>आचरण संहिता</h2>
+  <p>कुबेरनेट्स समुदाय मूल्यों में सम्मान और समावेश को महत्व देता, और सभी इंटरैक्शन में आचरण संहिता को लागू करता हैं।</p>
```
Comment (aj11anuj) [inline content/hi/community/_index.html:65]:
```suggestion
  <p>कुबेरनेट्स समुदाय मूल्यों में सम्मान और समावेश को महत्व देता है, और सभी इंटरैक्शन में आचरण संहिता को लागू करता हैं।</p>
```

File: content/hi/community/_index.html [hunk 15]
```diff
@@ -0,0 +1,175 @@
...
+  <p>यदि आप किसी इवेंट या मीटिंग, <a href="#slack">स्लैक</a> या किसी अन्य संचार माध्यम में आचरण संहिता का उल्लंघन करते हुए देखते हैं, तो <a href="mailto:conduct@kubernetes.io">conduct@kubernetes.io</a> पर कुबेरनेट्स आचार संहिता समिति से संपर्क करें। सभी रिपोर्ट को गोपनीय रखा जाता है। आप GitHub पर कुबेरनेट्स समुदाय रिपॉजिटरी में <a href="https://github.com/kubernetes/community/tree/master/committee-code-of-conduct">समिति के बारे में</a> पढ़ सकते हैं।</p>
+  <a href="/community/code-of-conduct/" class="community-cta-button">
+    <span class="community-cta">अधिक पढ़ें</span>
+  </a>
+</div>
+
+<div id="videos" class="community-section">
+  <h2>वीडियोज़</h2>
+
+  <p class="community-simple">कुबेरनेट्स यूट्यूब पर बहुत सारे वीडियो उपलब्ध हैं। विभिन्न विषयों के लिए सब्सक्राइब करें।</p>
```
Comment (aj11anuj) [inline content/hi/community/_index.html:75]:
```suggestion
  <p class="community-simple">कुबेरनेट्स के यूट्यूब चैनल पर बहुत सारे वीडियो उपलब्ध हैं। विभिन्न विषयों के लिए सब्सक्राइब करें।</p>
```

File: content/hi/community/_index.html [hunk 16]
```diff
@@ -0,0 +1,175 @@
...
+
+  <div class="container">
+    <div class="video youtube">
+      <iframe src="https://www.youtube.com/embed/videoseries?list=PL69nYSiGNLP3azFUvYJjGn45YbF6C-uIg" title="Monthly office hours" allow="camera 'none'; microphone 'none'; geolocation 'none'; fullscreen https://www.youtube.com/" ></iframe>
+      <a href="https://www.youtube.com/playlist?list=PL69nYSiGNLP3azFUvYJjGn45YbF6C-uIg">
+       <span class="videocta">मासिक ऑफिस घंटे देखें&nbsp;&#9654;</span>
```
Comment (aj11anuj) [inline content/hi/community/_index.html:81]:
```suggestion
       <span class="videocta">मासिक ऑफिस ऑवर्स देखें&nbsp;&#9654;</span>
```

File: content/hi/community/_index.html [hunk 17]
```diff
@@ -0,0 +1,175 @@
...
+      <a href="https://discuss.kubernetes.io/">समुदाय के फोरम्स&nbsp;&#9654;</a>
+      <p>विषय आधारित तकनीकी चर्चाएं जो दस्तावेज़, खोज और बहुत कुछ से जुड़ी होती हैं।</p>
+    </div>
+
+    <div id="twitter" class="community-resource">
+      <a href="https://twitter.com/kubernetesio">
+        <img src="/images/community/twitter.png" alt="Twitter">
+      </a>
+      <a href="https://twitter.com/kubernetesio">ट्विटर&nbsp;&#9654;</a>
+      <p><em>#कुबेरनेटेसियो</em></p>
```
Comment (aj11anuj) [inline content/hi/community/_index.html:120]:
```suggestion
      <p><em>#कुबेरनेट्स</em></p>
```

File: content/hi/community/_index.html [hunk 18]
```diff
@@ -0,0 +1,175 @@
...
+      <a href="https://github.com/kubernetes/kubernetes">गिटहब&nbsp;&#9654;</a>
+      <p>सभी प्रोजेक्ट और इश्यू ट्रैकिंग, साथ ही कोड भी होता है।</p>
+    </div>
+
+    <div id="server-fault" class="community-resource">
+      <a href="https://serverfault.com/questions/tagged/kubernetes">
+        <img src="/images/community/serverfault.png" alt="Server Fault">
+      </a>
+      <a href="https://serverfault.com/questions/tagged/kubernetes">सर्वर फॉल्ट&nbsp;&#9654;</a>
+      <p>सर्वर फॉल्ट पर Kubernetes से संबंधित चर्चा। सवाल पूछें या एक का जवाब दें।</p>
```
Comment (aj11anuj) [inline content/hi/community/_index.html:137]:
```suggestion
      <p>सर्वर फॉल्ट पर कुबेरनेट्स से संबंधित चर्चा। सवाल पूछें या एक का जवाब दें।</p>
```

File: content/hi/community/_index.html [hunk 19]
```diff
@@ -0,0 +1,175 @@
...
+      <details><summary><em>निमंत्रण चाहिए?</em></summary>
+        आमंत्रण के लिए  <a href="https://slack.k8s.io/">https://slack.k8s.io/</a>
+        पर जाएं।</details>
+    </div>
+  </div>
+</div>
+
+<div class="community-section" id="events">
+  <div class="container">
+   <h2>आगामी कार्यक्रम</h2>
```
Comment (aj11anuj) [inline content/hi/community/_index.html:155]:
```suggestion
   <h2>आगे आने वाले कार्यक्रम</h2>
```

File: content/hi/community/_index.html [hunk 20]
```diff
@@ -0,0 +1,175 @@
...
+   {{< upcoming-events >}}
+  </div>
+</div>
+
+<div class="community-section" id="meetups">
+  <h2>वैश्विक समुदाय</h2>
+  <p>
+    दुनिया भर में 150 से अधिक मीटअप हो चुके हैं और आगे बढ़ रहे हैं, जाएं और अपने स्थानीय क्यूब लोगों को खोजें। अगर आपके पास कोई नहीं है, तो स्वयं निर्माण करें।
```
Comment (aj11anuj) [inline content/hi/community/_index.html:163]:
```suggestion
    दुनिया भर में 150+ मीटअप हो चुके हैं और आगे बढ़ रहे हैं, जाएं और अपने स्थानीय कुबेरनेट्स कम्यूनिटी के लोगों को खोजें। अगर आपके पास कोई नहीं है, तो स्वयं निर्माण करें।
```

File: content/hi/community/_index.html [hunk 21]
```diff
@@ -0,0 +1,175 @@
...
+  </p>
+  <a href="https://www.meetup.com/topics/kubernetes/" class="community-cta-button">
+    <span class="community-cta">एक मुलाकात खोजें</span>
```
Comment (aj11anuj) [inline content/hi/community/_index.html:166]:
```suggestion
    <span class="community-cta">एक मीटअप खोजें</span>
```

File: content/hi/community/_index.html [hunk 22]
```diff
@@ -0,0 +1,175 @@
...
+  </a>
+</div>
+
+<div class="community-section community-frame" id="news">
+  <h2>नवीनतम समाचार</h2>
+  <div class="twittercol1">
+    <a class="twitter-timeline" data-tweet-limit="1" href="https://twitter.com/kubernetesio?ref_src=twsrc%5Etfw">कुबेरनेटेसियो द्वारा ट्वीट्स</a>
```
Comment (aj11anuj) [inline content/hi/community/_index.html:173]:
```suggestion
    <a class="twitter-timeline" data-tweet-limit="1" href="https://twitter.com/kubernetesio?ref_src=twsrc%5Etfw">कुबेरनेट्स द्वारा ट्वीट्स</a>
```

File: content/hi/community/_index.html [hunk 23]
```diff
@@ -0,0 +1,175 @@
...
+<img
+  id="banner"
+  srcset="/images/community/kubernetes-community-final-02.jpg 1500w, /images/community/kubernetes-community-02-mobile.jpg 900w"
+  sizes="(max-width: 900px) 900px, (max-width: 1920px) 1500px"
+  src="/images/community/kubernetes-community-final-02.jpg"
+  alt="कुबेरनेट्स सम्मेलन तस्वीर">
+
+<div class="community-section" id="introduction">
+  <p>कुबेरनेट्स समुदाय - उपयोगकर्ता, योगदानकर्ता और हम सभी ने साथ मिलकर बनाई हुई संस्कृति - इस ओपन सोर्स प्रोजेक्ट के उच्चाकांक्षी उदय के सबसे बड़े कारणों में से एक है। हमारी संस्कृति और मूल्य तब तक बढ़ते रहते हैं जब तक प्रोजेक्ट और हम इस पर काम करने के तरीकों में सुधार करते रहें। हम सभी संयुक्त रूप से प्रोजेक्ट और हम इस पर काम करने के तरीकों को लगातार सुधारने के लिए काम करते हैं।</p>
+  <p> हम वे लोग हैं जो मुद्दों और पुल अनुरोध फाइल करते हैं, एसआईजी बैठकों, कुबेरनेट्स मीटअप और क्यूबेकॉन में भाग लेते हैं, इसके अपनाने और नवाचार के लिए वकालत करते हैं,
```
Comment (aj11anuj) [inline content/hi/community/_index.html:16]:
```suggestion
  <p> हम वे लोग हैं जो मुद्दों और पुल अनुरोध फाइल करते हैं, SIG मीटअप, कुबेरनेट्स मीटअप और KubeCon में भाग लेते हैं, इसके अपनाने और नवाचार के लिए वकालत करते हैं।
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
Work on these changes

**Comment by niranjandarshann** [comment]
@harish343  
Could you please squash your commits: https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits

**Comment by niranjandarshann** [comment]
@harish343  Are you currently working on this issue? Please make the suggested changes and address the comments. I am seeing this PR is inactive from long ago.

**Comment by divya-mohan0209** [comment]
Hi @harish343 , conversations on this PR have been going on for > 1 year & despite reminders, there has been no progress. Therefore, I shall be closing it. If you have the bandwidth to resume work on it, please feel free to reopen the PR.
/close

**Pull request closed without merging**